modular_skyrat/modules/customization/modules/language/calcic.dm

```diff
@@ -0,0 +1,2 @@
+/datum/language/calcic
+	secret = TRUE
```

Comment:
this is a language used by an actively selectable species(plasmamen), this needs to be selectable

Reply:
Wrong PR